src/bins/driver.rs

```diff
@@ -14,19 +14,27 @@ use rustc_session::config::ErrorOutputType;
 use std::convert::TryFrom;
 
 fn main() {
+    // We use the default hook for internal compiler errors (ICE).
     rustc_driver::install_ice_hook();
+    // This method call is used to set up rust logging.
     rustc_driver::init_rustc_env_logger();
 
+    // Gets the command line arguments sent to the program and adds them to a vector.
     let mut rustc_args: Vec<String> = vec![];
 
     for arg in std::env::args() {
         rustc_args.push(arg);
     }
 
+    // We use our function to run the compiler with the supplied arguements and the callbacks
+    // which we would like the rust compiler to make during the compilation process.
     run_compiler(rustc_args, &mut TaintCompilerCallbacks)
 }
 
+// Our wrapper function for running the rust compiler
 fn run_compiler(mut args: Vec<String>, callbacks: &mut (dyn rustc_driver::Callbacks + Send)) -> ! {
+    // We first make sure that the sysroot argument to our executable is set.
+    // The sysroot should be pointing to the toolchain named in the "rust-toolchain" file.
```

Comment:
We're just checking if we can locate the sysroot, and if we can, we add it as an argument to rustc explicitly. Otherwise we just use the default.

Reply:
Okay, der ændrer jeg også lige formuleringen. 
Hvad vil der ske ved at vi bruger den standard sysroot?